livy/src/main/java/org/apache/zeppelin/livy/LivySparkRInterpreter.java

```diff
@@ -43,4 +43,16 @@ public LivySparkRInterpreter(Properties property) {
   public String getSessionKind() {
     return "sparkr";
   }
+
+  @Override
+  protected String extractAppId() throws LivyException {
+    //TODO(zjffdu) depends on SparkR
+    return null;
+  }
+
+  @Override
+  protected String extractWebUIAddress() throws LivyException {
+    //TODO(zjffdu) depends on SparkR
+    return null;
```

Comment:
Thanks for the hint. Livy interpreter need to support both spark1 and spark2. Considering the complexity and time limitation (I will take vacation soon), so I would do it in a next PR.